src/vendor.browser.ts

```diff
@@ -13,7 +13,7 @@ import 'rxjs/add/operator/mergeMap';
 
 // 3rd Party Libraries
 import 'jquery';
-import 'bootstrap';
 import 'lodash';
 import 'd3';
 import 'd3-tip';
+import 'ng2-bootstrap';
```

Comment:
Oh shoot. Sorry, I missed the use of that component yesterday while scanning through. Using ng2-bootstrap sounds good, removing the bootstrap dependency is also good. 

Is this import here in vendor.browser.ts necessary? I would think the piecemeal imports in each of our source files would be enough.